### ISSUE TITLE
Night Mode - All lights off

### ISSUE THREAD

**Issue by NasaEagle** [bambulab/BambuStudio#4037]
I have my X1C printer with AMS in the room I sleep in, and at times, the bright lights from the logo and the AMS bother me at times (I know the AMS lights aren't super bright, but when the room is pitch black it can be frustrating).

My alternative at the moment is by putting a piece of electrical tape over the logo, but the light still leaks over the top of the tool head.

It would be nice to add a setting to turn off all the lights (cabin, tool head logo, AMS). Another possible solution could be to turn off all the lights when the front control backlight turns off, and all the lights could turn on when the panel is touched or reactivated (this could be controlled by a button in the settings page). 

This would be great for those who have their printers in their room, or who wish to not have the lights running for the duration of a print.

Thanks!

**Comment by zhimin-zeng-bambulab** [comment]
![image](https://github.com/bambulab/BambuStudio/assets/120697589/2967255e-c1c0-4267-a8f9-182fb3ed1cf2)
@NasaEagle  You can click this button on the device page to control the light switch. Does this meet your needs?

**Comment by NasaEagle** [comment]
No, it’s more about being able to control the lights on the tool head and the AMS.

Carter Stengel

On May 15, 2024, at 1:12 AM, zhimin-zeng-bambulab ***@***.***> wrote:

﻿

image.png (view on web)<https://github.com/bambulab/BambuStudio/assets/120697589/2967255e-c1c0-4267-a8f9-182fb3ed1cf2>
@NasaEagle<https://github.com/NasaEagle> You can click this button on the device page to control the light switch. Does this meet your needs?

**Comment by MrDB42** [comment]
New version of BBS should have added the following to the bottom of "Machine end G-code"
M960 S5 P0 ; turn off logo lamp

BUT... My issue is that sometimes the chamber light will turn on at the end of the print. I haven't figured out the pattern yet. Would _love_ to know the G-code to control the chamber light.

**Comment by markwalkom** [comment]
I wanted to add a +1 to this because even when you turn off a printer, or it goes to sleep mode, the AMS lights *don't* turn off. And it's kinda annoying.

**Comment by Frozander** [comment]
Same here. I wouldn't mind having 2 buttons, even under some submenus to turn off tool and AMS lights. (Especially the tool light)

**Comment by picard102** [comment]
Yes, the tool head having to UI to control it seems wild.
